docker-angor-api/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+volumes:
+  blockchain:
+    name: btc-data
+
+services:
+  node:
+    container_name: btc-node
+    image: eugenenostrdev/bitcoin_signet:amd64
```

Comment:
this is compiled form the same signet as our other network right?
you just pushed the image to docker hub?